src/DotVVM.Framework/Routing/DotvvmRouteTable.cs

```diff
@@ -104,6 +104,21 @@ public void Add(string routeName, string url, string virtualPath, object default
             Add(group?.RouteNamePrefix + routeName, new DotvvmRoute(group?.UrlPrefix + url, group?.VirtualPathPrefix + virtualPath, defaultValues, presenterFactory, configuration));
         }
 
+        /// <summary>
+        /// Adds the specified route name.
+        /// </summary>
+        /// <typeparam name="T">Type of presenter</typeparam>
+        /// <param name="routeName">Name of the route.</param>
+        /// <param name="url">The URL.</param>
+        /// <param name="virtualPath">The virtual path of the Dothtml file.</param>
+        /// <param name="defaultValues">The default values.</param>
```

Comment:
When I look at the parameter name and the comment, I don't see any single bit of information that the comment gives me...